doc/user/content/administration/billing.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Usage & Billing"
+description: "Provides an overview of various factors that can affect cost."
+menu:
+  main:
+    parent: "administration"
+    weight: 10
+---
+
+From the [console](https://console.materialize.com/) (`Admin` > `Usage &
+Billing`), administrators can access their invoice. The invoice provides
+Compute and Storage usage and cost information.
+
+Various factors can affect your costs, including (but not limited to):
+
+- Each [replica for a cluster](/sql/create-cluster/#credit-usage). To see the
+  number of replicas for your clusters, click on `Clusters` in the console,
+
+- Creating a [materialized view](/concepts/views/#materialized-views) as
```

Comment:
Figured I'd list the creating mat view and indexes, since I think these are things where people know but forget (since after you create them, they might not be thinking about them).